src/components/course/CourseSidebarPrice.jsx

```diff
@@ -98,8 +98,9 @@ const CourseSidebarPrice = () => {
     );
   }
 
-  const discountedPriceMessage = userSubsidyApplicableToCourse?.subsidyType
-   === ENTERPRISE_OFFER_SUBSIDY_TYPE ? COVERED_BY_ENTERPRISE_OFFER_MESSAGE : `Sponsored by ${enterpriseConfig.name}`;
+  const learnerCreditSubsidyTypes = [ENTERPRISE_OFFER_SUBSIDY_TYPE, LEARNER_CREDIT_SUBSIDY_TYPE];
+  const shouldShowLearnerCreditMessage = learnerCreditSubsidyTypes.includes(userSubsidyApplicableToCourse?.subsidyType);
+  const discountedPriceMessage = shouldShowLearnerCreditMessage ? COVERED_BY_ENTERPRISE_OFFER_MESSAGE : `Sponsored by ${enterpriseConfig.name}`;
```

Comment:
Learner credit messaging should show for both legacy enterprise offers and new EMET learner credit.

<img width="386" alt="image" src="https://user-images.githubusercontent.com/2828721/234416940-7974a91d-4206-415f-b5ca-1f2ba9b197cb.png">